app/FAIRshakeWeb/templates/project_evaluated_resources.html

```diff
@@ -0,0 +1,110 @@
+{% extends "base.html" %}
+
+{% block header %}
+    {{ super() }}
+
+    {% block style %}
+        {{ super() }}
+        <link rel="stylesheet" href="{{ url_for('static', filename='style/project_resources.css') }}" />
+    {% endblock %}
+{% endblock %}
+
+{% block content %}
+    <script>
+        function project_resources(){
+            location.href = "/project/{{ project.id }}/resources";
+        }
+    </script>
+    <div class="container-fluid">
+        <div class="heading">
+            <div class="title">
+                <h2>{{ project.name }}<small> - Resources List</small></h2>
+                <h4>{{ project.description }} </h4>
+            </div>
+{#            {% if current_user.is_authenticated %}#}
+                <div class="btn-group" id="listgrpbtns" data-toggle="buttons">
+                    <label class="btn btn-default listgrpbtn" onclick="project_resources()">
```

Comment:
This can just be a link, I would try to refrain from writing javascript as much as possible unless it is essential.